### PR TITLE
Update deployment status

### PR DESCRIPTION
**Pull request by zhouying7780**
@liangxia Could you please help review and merge , my prior pr is wrong , this will fix it. 
runner log is here :
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/860597/console

### CODE REVIEW

File: features/step_definitions/deployment.rb [hunk 1]
```diff
@@ -62,7 +62,10 @@
   success = wait_for(seconds) do
     rs_name != deplmnt.current_replica_set(user: user, cached: false).name
   end
-  raise 'expected replica set name change not reached within timeout' unless success
+  unless success
+    logger.error(user.cli_exec(:describe, resource_name: "deployment/#{d_name}"))
+    raise "Deployment did not become ready"
```
Comment (liangxia) [inline features/step_definitions/deployment.rb:67]: Maybe we need to update the msg to reflect the real status ?

Comment (zhouying7780) [reply features/step_definitions/deployment.rb:67]: @liangxia we could see the deployment status from the output of `oc describe` command .

Comment (zhouying7780) [reply features/step_definitions/deployment.rb:67]: @liangxia Any other suggestion we need to update ?

Comment (liangxia) [reply features/step_definitions/deployment.rb:67]: I mean the msg in **raise** is not reflecting the real situation.

**Comment by zhouying7780** [comment]
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/860608/console

@liangxia So sorry , I got you , I correct my fault . please help review , thanks .

**Pull request closed without merging**